release-notes-3.5.2.md

```diff
@@ -4,5 +4,6 @@
 ${version-number}
 
 #### New Features
+- US355271: Dependency updates
```

Comment:
Not sure if this should be under New Features.  Also check what should be written under Known Issues, I suspect "none", or maybe remove the section-check some previous release notes in guthub.